content/questions/admin/question-004.md

```diff
@@ -0,0 +1,15 @@
+---
+question: "Which of the following issues can GitHub Support help resolve? (Choose three.)"
+archetype: "questions"
+title: "Question 004"
+draft: false
+---
+
+> https://docs.github.com/en/enterprise-server@3.11/support/learning-about-github-support/about-github-support#scope-of-support
+- [x] Installing and using GitHub Enterprise Server
+- [x] Identifying and verifying the causes of suspected errors
```

Comment:
Either remove that answer and just leave 3 correct answers because the others are clear
or
Update the title to (**Choose four)** 

I leave the choice to you

Reply:
That exact sentence is on the official docs tho.

Reply:
That exact sentence is in the official docs tho.

Reply:
Ok, so change the title to Choose four